Que-Me-Pongo/src/test/java/test/modelo/utils/UtilsTests.java

```diff
@@ -11,16 +13,20 @@
 import modelo.clases.Guardarropas;
 import modelo.clases.SuscripcionPremium;
 import modelo.clases.Usuario;
-import utils.Utils;
+import utils.emailSender;
+
 
 @DisplayName("Tests para las funciones de utilidad")
 public class UtilsTests extends Application {
 
+	emailSender notificador = new emailSender();
```

Comment:
Please change this name no notification instead of notificador.

Reply:
Listo